apps/api/src/types/user/dto/create-user.input.ts

```diff
@@ -15,4 +15,5 @@ export class CreateUserInput extends OmitType(User, ["id", "joined"], InputType)
     @MinLength(8)
     @IsOptional()
     password: string | null;
+    username: any;
```

Comment:
The `CreateUserInput` type extends the `User` type, which already has a `username` field, so it doesn't need to be set here explicitly.